pkg/mocks/provider.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package mocks
+
+import "github.com/hyperledger/aries-framework-go/pkg/wallet"
+
+// Provider mocks provider needed for did exchange service initialization
+type Provider struct {
```

Comment:
@fqutishat @troyronda Should we expose mocks ? Currently we have [transport mocks](https://github.com/hyperledger/aries-framework-go/tree/master/pkg/internal/didcomm/transport/mock) in internal package and others in test files.

Reply:
Should be internal.

Reply:
I move it